atom/renderer/api/atom_api_spell_check_client.h

```diff
@@ -5,6 +5,7 @@
 #ifndef ATOM_RENDERER_API_ATOM_API_SPELL_CHECK_CLIENT_H_
 #define ATOM_RENDERER_API_ATOM_API_SPELL_CHECK_CLIENT_H_
 
+#include <map>
```

Comment:
There's no std::map in this header, so this shouldn't be #included here

Reply:
Updated, leftover from my previous test implementation.